changelog/fragments/swap-readiness-liveness-paths-4541.yaml

```
@@ -0,0 +1,30 @@
+# issue #4541
+entries:
+  - description: >
+      For Ansible/Helm-based operators, fix swapped readinessProbe/livenessProbe in manager for projects
+      created with operators-sdk >= v1.4.0.
```

The version is implied by the release notes this fragment will be a part of.
```suggestion
      For Ansible/Helm-based operators, fix swapped readinessProbe/livenessProbe in manager
```